connectors/sources/gmail.py

```diff
@@ -24,6 +25,12 @@
 )
 from connectors.utils import base64url_to_base64, iso_utc
 
+CUSTOMER_ID_LABEL = "Google customer id"
+
+SUBJECT_LABEL = "Subject"
+
+SERVICE_ACCOUNT_CREDENTIALS_LABEL = "GMail service account JSON"
+
```

Comment:
Why make it constants? It makes it harder to copy-paste configuration into JSON, for example, and does not add any value IMO

Reply:
Because I'm using these values at two places in the code, so if it's changed it's reflected at both places.

Reply:
Ah, overlooked it